docs/source/hosted_applications/NREL_APPS/Solar_Forecasting.rst

```diff
@@ -0,0 +1,66 @@
+Solar Forecasting Application
+---------------------------------------------------------------
```

Comment:
Headers only to the end of the above line

Reply:
Okay Fixed.